pkg/cmd/scan.go

```diff
@@ -154,6 +157,14 @@ func NewScanCmd() *cobra.Command {
 		"Includes cloud provider service-linked roles (disabled by default)",
 	)
 
+	configDir, _ := homedir.Dir()
+	fl.String(
+		"config-dir",
+		configDir,
+		".driftctl path to use.\n"+
+			"Default is home directory.\n",
```

Comment:
I find this sentence really confusing.

I'd rather read: 
`Directory path that driftctl uses for configuration`
`Default is the home directory`

Reply:
FYI Cobra already handles default values in help message. You shoudn't have to write it manually.